magprime/templates/emails/precon_faqs.html

```diff
@@ -7,24 +7,40 @@
+</p>
+<p>
+    <b> Will additional passes be sold at the door?</b><br />
+    Yes, but there is a possibility of us selling out, so check our website for any ticket availability alerts. Refer to <a href="http://magfest.org/announcement/important-announcement-regarding-badges/" target="_blank"><b>this announcement</b></a> for pricing details.
+</p>
+<p>
+    <b>Which registration line do I use?</b>
+    <ul>
+        <li>Staff, Supporters, Guests, Vendors, Indie showcase exhibitors, and Panelists share the same line. Look for the signage inside Expo Hall E.</li>
+        <li>All other pre-registered attendees (including those who purchased group registrations, press, or those who registered at-door) should use the "Pre-Registration Badge Pickup" line inside Expo Hall E.</li>
+        <li>If you are not already registered for a pass, you can <a href="http://magfest.org/register" target="_blank"><b>purchase one online</b></a>, or use the self-service registration and payment stations located on the left side of Expo Hall E at the "At-Door Registration" sign. Cash and Credit are accepted. Once you are registered and paid, proceed to the badge pickup line inside Expo Hall E.</li>
+    </ul>
+</p>
+<p>
+    <b>Where can I pick up swag that I pre-ordered, such as a t-shirt or supporter bundle?</b><br />
+    After you've picked up your pass, you can pick up your swag at the MAGFest merch booth. The merch booth is located at the front of the marketplace in Expo Hall B. Follow the red line in <a href="http://magfest.org/static/images/mag2017_map_to_merch.png" target="_blank"><b>this map</b></a> to help find your way to the merch booth from the registration area.
+</p>
+<p>
+    <b>Where can I view a schedule of events?</b><br />
+    There are multiple ways to view our schedule:</p>
+    <ul>
+        <li><i>Using the Guidebook app or website (desktop or mobile)</i>: Go to the  <a href="http://schedule.magfest.org/" target="_blank">Guidebook website</a> to view the guidebook in your web browser or download the mobile app. Make sure to check for guidebook updates periodically through the weekend, as we may make schedule changes.</li>
+        <li><i>Print your own schedule</i>: We have 11x17 a double-sided formatted <a href="http://magfest.org/static/files/M2017_Schedule.pdf" target="_blank">print schedule</a> that will be available in our registration area. Since schedules are posted outside individual rooms, and are also available online, we will only have limited quantities of the printed schedule available. To be safe, please print your own copy of the schedule if you rely on having a hard copy handy.</li>
+    </ul>
+</p>
+{% if attendee.is_transferable %}
```

Comment:
I would slightly prefer that this question exist regardless of transfer status, but have different answers depending on transfer status, e.g., providing the link if they're transferable or saying to contact Registration otherwise.